plugins/de.cau.cs.kieler.kgraph.text.ui/src/de/cau/cs/kieler/kgraph/text/ui/contentassist/KGraphProposalProvider.java

```diff
@@ -471,13 +472,19 @@ private void valueProposal(final ContentAssistContext context,
                         break;
                     case OBJECT:
                         try {
+                        
                             proposal = "\""
-                                    + optionData.getOptionClass().newInstance().toString()
+                                    + optionData.getOptionClass().getConstructor().newInstance().toString()
                                     + "\"";
+                        
                         } catch (final InstantiationException e) {
                             proposal = "\"\"";
                         } catch (final IllegalAccessException e) {
                             proposal = "\"\"";
+                        }catch (IllegalArgumentException | InvocationTargetException
+                                | NoSuchMethodException | SecurityException e) {
+                            proposal = "\"\"";
+                            e.printStackTrace();
```

Comment:
Why does this exception have a different handling than the others?

Reply:
The main reason is that the InvocationTargetException was previously a different  RuntimeException and therefore not caught. 
The NoSuchMethodeExcpetion would previously fall under the InstantiationException and should probably be handled as such.
The IllegalArgumentException should not be possible as we request a nullary Constructor and then call the constructor without arguments, as such it shouldn't make a difference. 
I don't know why I am explicitly handling the SecurityException.

Reply:
As discussed only the InvocationTargetException is handled separately now